src/OrgUsageDashboard.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <div class="org-usage-dashboard">
+    <Message class="component" type="alert-info" message="<p><strong>This feature is currently in beta.</strong> We are actively refining the functionality and collecting feedback. If you have any questions please <a href='#'>contact us</a>.<p>" />
```

Comment:
@MaksymShokin How is the **Contact us** link going to open the live chat feature? It was mentioned in the specification.

Reply:
@tonytlwu 
My bad. Missed that. Fixed

Reply:
@tonytlwu Type also fixed